target/snitch_cluster/README.md

```diff
@@ -1,24 +1,60 @@
-# Snitch cluster target
+# SNAX Cluster Target
 
+## Details
 The Snitch cluster target (`target/snitch_cluster`) is a simple RTL testbench
-around a Snitch cluster. The cluster can be configured using a config file. By default, the config file which will be picked up is `target/snitch_cluster/cfg/default.hsjon`.
+around a SNAX cluster. It is still the same as the Snitch cluster but with SNAX modifications. The cluster can be configured using a config file. By default, the config file which will be picked up is `target/snitch_cluster/cfg/default.hsjon`.
```

Comment:
Currently, we are using lru.hjson as default? I thought if no lru.hjson, then use defalut.hjson.

Reply:
No no, `lru.hjson` becomes the main `.hjson` that the make file uses. So what it does is to copy the `default.hjson` or any of the `custom.hjson` into the lru.hjson. But the main default is just the `default.hjson` if that makes sense 👯